pandas/tseries/resample.py

```diff
@@ -499,12 +499,12 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
                                   limit_direction=limit_direction,
                                   downcast=downcast, **kwargs)
 
-    def asfreq(self):
+    def asfreq(self, **kwargs):
```

Comment:
good point. i had thought it was cleaner since the arguments just get passed to _apply through _upsample. But now I'm thinking about it more and it makes sense to list the arguments here --- it is also more consistent with backfill and pad